### PR TITLE
Room page routing

### PR DESCRIPTION
**Pull request by JamieZafar**

### CODE REVIEW

**Comment by dwybourn** [comment]
I'd expect clicking on the room to actually take me to the room.

File: src/client/index.js [hunk 1]
```diff
@@ -2,21 +2,25 @@ import React from 'react';
...
-render(
-    <AppContainer>
-        <Provider store={store}>
-            <App />
-        </Provider>
-    </AppContainer>,
-    app
-);
+render((
+    <Provider store={store}>
```
Comment (dwybourn) [inline src/client/index.js]: Why has the AppContainer been removed?

**Comment by thullSL** [comment]
@dwybourn Clicking the room does take you to the room does it not?

**Comment by JamieZafar** [comment]
Clicking on the room does take you to the room, swer on me nan

**Comment by dwybourn** [comment]
Apologies, I was on the wrong branch.

File: src/client/lobby/room-page.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
```
Comment (dwybourn) [inline src/client/lobby/room-page.js]: The room scripts should be in their own folder called rooms.

**Comment by JamieZafar** [comment]
:ship: ?

File: src/client/rooms/room-page.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+export default class RoomPage extends Component {
+    render() {
+        return (
+            <div className="col-lg-4 text-center">
+            <h2>{`Room ${this.props.params.id}`}</h2>
```
Comment (thullSL) [inline src/client/rooms/room-page.js]: This'll look weird for most cool names for a room, 
`Room the friday room`
`Room tom's room`
`Room library`

Can we get a colon at least `Room: the commit room`

Comment (JamieZafar) [inline src/client/rooms/room-page.js]: "cool"

File: src/client/lobby/lobby.js [hunk 4]
```diff
@@ -47,6 +50,10 @@ export class Lobby extends Component {
     }
 };
 
+function redirectToRoom(arg){
+    browserHistory.push(`/room/${arg}`);
```
Comment (thullSL) [inline src/client/lobby/lobby.js]: Does this add the page to history as you is going to it? Isn't it supposed to be the previous page that's added?

Comment (JamieZafar) [inline src/client/lobby/lobby.js]: i think its a react-router thing, its what moves the user to a different page

Comment (thullSL) [inline src/client/lobby/lobby.js]: :+1:

**Comment by thullSL** [comment]
:shipit:

**Pull request merged**